src/Raven.Server/Storage/Schema/Updates/Documents/42017/From41016.cs

```diff
@@ -619,6 +624,17 @@ private static BlittableJsonReaderObject WriteNewCountersDocument(DocumentsOpera
 
                     builder.WriteObjectEnd();
 
+                    builder.WritePropertyName(CountersStorage.CounterNames);
```

Comment:
The only method that is shared between `From41016` and `From41017` is `ReadDbId()`
And yes, I intoduced some changes in `From41016` and upgrading from 4.1.x to 5.0 now works 
and is tested - unskipped `CanMigrateLegacyCounters`, `CanMigrateLegacyCountersWithMultipleDbIds`, `CanMigrateTablesWithCounterWord` tests

Reply:
But it means that when 42017.cs is executed and migration was done from 41016, it will already produce the CounterNames property. Is this something that we would want? It means that 42017.cs will encounter them, but it shouldn't?

Reply:
reverted changed in `From41016`, redundant work